.codeclimate.yml

```diff
@@ -0,0 +1,27 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+      - javascript
+      - python
+      - php
+  eslint:
+    enabled: false
+    checks: 
+      import/extensions:
+        enabled: false
```

Comment:
I have no idea how this works, but isn't this disabling twice? eslint, then import/extensions?

Reply:
It is, but I left it in as a reminder that import/extensions check appears to be buggy, so even if we decide to re-enable eslint, we should keep that one switched off.